cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/TPnodeEquipments.java

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.cgmes.conversion.elements.hvdc;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.powsybl.cgmes.model.CgmesDcTerminal;
+import com.powsybl.cgmes.model.CgmesModel;
+import com.powsybl.cgmes.model.CgmesNames;
+import com.powsybl.cgmes.model.CgmesTerminal;
+import com.powsybl.triplestore.api.PropertyBag;
+import com.powsybl.triplestore.api.PropertyBags;
+
+/**
+ *
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+class TPnodeEquipments {
+
+    enum EquipmentType {
+        TRANSFORMER, AC_DC_CONVERTER, DC_LINE_SEGMENT
+    }
+
+    private final Map<String, List<TPnodeEquipment>> nodeEquipments;
+
+    TPnodeEquipments(CgmesModel cgmesModel, Adjacency adjacency) {
+        nodeEquipments = new HashMap<>();
+
+        cgmesModel.dcLineSegments().forEach(dcls -> computeDcLineSegment(cgmesModel, adjacency, dcls));
+
+        AcDcConverterNodes acDcConverterNodes = new AcDcConverterNodes(cgmesModel);
+
+        acDcConverterNodes.getConverterNodes().entrySet()
+            .forEach(entry -> addEquipment(adjacency, entry.getValue().id, entry.getValue().acTopologicalNode,
```

Comment:
You only use the values of entries here, can you just use `getConverterNodes().values().forEach(...)`?

Reply:
Done